### ISSUE TITLE
Schema: Breadcrumbs missing field "position" in `itemListElement` SlugHub INC1953747

### ISSUE THREAD

**Issue by Herm71** [ucsc/ucsc-2022#330]
### Description

Google Search Console has flagged **Breadcrumbs** as missing the field "position" in the `itemListElement`. See SlugHub ticket [INC1953747](https://slughub.ucsc.edu/now/nav/ui/classic/params/target/incident.do%3Fsys_id%3Dbbac27351b46ca90bc0f54e4604bcb86%26sysparm_stack%3D%26sysparm_view%3D).

### Step-by-step reproduction instructions

This error is shown in the Google Search Console for any UCSC Themed WordPress site under the Breadcrumbs side menu.

### Screenshots, screen recording, code snippet

Google Search Console Error:
![missing-field](https://github.com/ucsc/ucsc-2022/assets/1000543/8c788fa5-9735-4f44-9d08-3e76ff6fc3ab)

![position](https://github.com/ucsc/ucsc-2022/assets/1000543/afbfd92b-ed3e-4611-bc85-7b3be5675542)

### Environment info

Any UCSC site with Breadcrumbs.

### Please confirm that you have searched existing issues in the repo.

Yes

**Comment by Herm71** [comment]
As of June 27th, 2024, the Google Search Console is still showing the error referenced above.
